Tools/Linux_HAL_Essentials/pru/aiopru/RcAioPRU_BBBLUE_bin.h

```diff
@@ -0,0 +1,216 @@
+
+
+/* This file contains the PRU instructions in a C array which are to  */
+/* be downloaded from the host CPU to the PRU instruction memory.     */
+/* This file is generated by the PRU assembler.                       */
+
+const unsigned int PRUcode[] =  {
```

Comment:
The preferred support model by the Linux community moving forward for loading firmware on the PRU is remote proc, not UIO. All the same simple shared-memory communication is possible, but it moves the firmware loader from userspace into the Linux kernel. Blue will ship with remote proc enabled, so the use of UIO would require changes to boot files. Firmware file names can be updated with device tree overlays and reset/reload can be performed by unbinding and binding the driver.